### PR TITLE
Add InvalidateTileAir Transform extension method

### PR DESCRIPTION
**Pull request by DrSmugleaf**

### CODE REVIEW

File: Content.Server/Atmos/AtmosHelpers.cs [hunk 1]
```diff
@@ -11,18 +12,18 @@ namespace Content.Server.Atmos
...
     {
-        public static TileAtmosphere? GetTileAtmosphere(this EntityCoordinates coordinates, IEntityManager? entityManager = null)
+        public static TileAtmosphere GetTileAtmosphere(this EntityCoordinates coordinates, IEntityManager? entityManager = null)
         {
             entityManager ??= IoCManager.Resolve<IEntityManager>();
 
             var gridAtmos = EntitySystem.Get<AtmosphereSystem>().GetGridAtmosphere(coordinates.GetGridId(entityManager));
 
-            return gridAtmos?.GetTile(coordinates);
+            return gridAtmos.GetTile(coordinates);
```
Comment (metalgearsloth) [inline Content.Server/Atmos/AtmosHelpers.cs:21]: What happens if the grid doesn't have atmos?

Comment (DrSmugleaf) [reply Content.Server/Atmos/AtmosHelpers.cs:21]: It returns a SpaceAtmosphereComponent.

https://discord.com/channels/310555209753690112/560845886263918612/783748700588867614

**Review by PJB3005 — APPROVED** [review]

**Review by Zumorica — APPROVED** [review]
A single invalidate method that takes in ITransformComponent is a bit odd, but okay

**Comment by DrSmugleaf** [comment]
> A single invalidate method that takes in ITransformComponent is a bit odd, but okay

Added one for EntityCoordinates too.

**Comment by Zumorica** [comment]
Appreciated.

**Pull request merged**